server/src/workspace/workspace-sync-metadata/standard-objects/company.object-metadata.ts

```diff
@@ -124,7 +131,8 @@ export class CompanyObjectMetadata extends BaseObjectMetadata {
     type: RelationMetadataType.ONE_TO_MANY,
     objectName: 'activityTarget',
   })
-  activityTargets: object[];
+  @IsNullable()
```

Comment:
this seems wrong to me to add IsNullable on OneToMany, is this because it's not taken into account on GraphQL side?

Reply:
@charlesBochet I agree but this was like this before which is why it's failing now. Without IsNullable the field will be required on the input apparently @charlesBochet

Reply:
Ok, let's leave it like this, but can you create a ticket about isNullable and relations. We should take it next week with @magrinj

Reply:
Ok 👍